src/main/target/KAKUTEF4/target.mk

```diff
@@ -6,4 +6,6 @@ TARGET_SRC = \
             drivers/barometer_bmp280.c \
             drivers/barometer_ms5611.c \
             drivers/compass_ak8963.c \
-            drivers/compass_hmc5883l.c 
\ No newline at end of file
+            drivers/compass_hmc5883l.c \
+            drivers/max7456.c \
+            io/osd.c 
```

Comment:
io/osd.c is standard for HIGHEND_SRC (look at Makefile). There is also a general rule that only drivers go into target.mk.